parlai/core/torch_ranker_agent.py

```diff
@@ -885,7 +900,7 @@ def encode_candidates(self, padded_cands):
             '--encode-candidate-vecs True.'
         )
 
-    def _make_candidate_encs(self, vecs, path):
```

Comment:
was the purpose of `path` to be able to save the encodings and reload them?

Reply:
yes but that's currently handled by a different function.